src/YesSql.Provider.PostgreSql/PostgreSqlDialect.cs

```diff
@@ -149,6 +150,25 @@ public override string GetTypeName(DbType dbType, int? length, byte? precision,
             throw new Exception("DbType not found for: " + dbType);
         }
 
+        public override string FormatKeyName(string name)
+        {
+            if (name.Length >= 63)
```

Comment:
What is the actual max length, so we can understand where the 63 comes from.

Reply:
Added links to the docs